### ISSUE TITLE
Calculations for J

### ISSUE THREAD

**Issue by ketanshende** [HritikKumarSharma/Housing-Prices-Prediction-Linear-Regression#1]
It gives Error while calculating for J.

J=(1/(2*m))*sum((x*theta - y).^2,'all');

Something is wrong when you add 'all'

**Comment by HritikKumarSharma** [comment]
No, 'all' will add all the matrix elements and there is no need to use 'all' option. Without 'all' we can get the right results.

**Comment by HritikKumarSharma** [comment]
Read the size funtions documentation 

>> help size
